### DISCUSSION TITLE
Oh My Posh Error:  ReadMapCB: expect { or n, but found f, error found in #1 byte of ...|flag provid|..., bigger context ...|flag provided but not defined: -config-format Usag|...

### DISCUSSION THREAD

**Discussion by Karma-Tsering** [JanDeDobbeleer/oh-my-posh#1806] [Troubleshoot]
I am get this error after running below command in my powershell.

### Export-PoshTheme -FilePath "~/larserikfinholt.omp.json" -Format json

My aim is to change the theme from default jandedobbelee.omp.json to larserikfinholt.omp.json and I edit the same in $profile as bellow

### oh-my-posh --init --shell pwsh --config ~/larserikfinholt.omp.json | Invoke-Expression

Unfortunately, not only the error pop up but also lost my default theme and for each command, this line of error keeps popping up.

How can I get rid of this warning or error and get my desire theme.
Thanks lots devs

**Comment by JanDeDobbeleer** [comment]
@kar-cee this is a bug, fix incoming

**Reply by Karma-Tsering** [reply]
Thank you so much for your prompt response.
But I want my default theme by jandedobbeleer back and are there any ways to get back?

**Reply by JanDeDobbeleer** [reply]
I don't understand what went wrong tbh, nothing should be removed.
